src/main/webapp/form.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Student Recommendations</title>
+    <title>StudRecs - Form</title>
```

Comment:
I think maybe this title could be improved? "StudRecs" isn't a shortening we're using - it might be best to use the full wording.